.github/workflows/azure-static-web-apps-thankful-mud-0addb6c00.yml

```diff
@@ -0,0 +1,47 @@
+name: Azure Static Web Apps CI/CD

```

Comment:
is the file name meant to be random?

Reply:
The random string refers to the container instances it refers to, it has been removed. The secret name in yml has been renamed to be more meaningful as well.